changelog/issue-2962.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: silent
+reference: issue 2962
+---
+taskcluster-lib-postgres now supports `cryptoKeys` which allow for encrypting columns of a table.
```

Comment:
@helfi92 we'll need another changelog for the table change, with type `major` due to the newly-required config for the worker-manager service.